packages/react-i18n/src/test/i18n.test.ts

```diff
@@ -605,31 +605,39 @@ describe('I18n', () => {
     const input = 123456.7891;
 
     it('handles number with period decimal symbol', () => {
-      formatNumber.mockImplementationOnce(() => '1.1');
+      formatNumber.mockImplementationOnce(() => '123,456.7');
 
       const formatted = '123,456.7891';
 
       expect(i18n.unformatNumber(formatted)).toBe(input.toString());
     });
 
     it('handles number with comma decimal symbol', () => {
-      formatNumber.mockImplementationOnce(() => '1,1');
+      formatNumber.mockImplementationOnce(() => '123.456,7');
 
       const formatted = '123.456,7891';
 
       expect(i18n.unformatNumber(formatted)).toBe(input.toString());
     });
 
+    it('handles number with space as the thousand symbol', () => {
+      formatNumber.mockImplementationOnce(() => '123 456,7');
```

Comment:
Could not find any examples of languages using a space for the thousand separator and a period for the decimal.
See: https://docs.oracle.com/cd/E19455-01/806-0169/overview-9/index.html